policies/project-director-election-process.md

```diff
@@ -0,0 +1,76 @@
+    * **Goals of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944)). The role description must contain the following content at a minimum:
+    * Goals
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * The nomination process begins with an announcement from the facilitator chosen in the Setup phase. Council representatives should help publicize this announcement to their teams as well.
+    * The announcement will specify a date by which nominations must be submitted. This date will be no less than two weeks from the date the announcement goes out, and no more than four weeks.
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+      * Nominations do not have to be members of the Council representative's respective team but must be project members. 
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+    * The facilitator may specify a minimum or maximum time period 
+* Share nominations (*Nominations from all teams will be pooled together. In other words, teams do not have to nominate a set of directors as a whole. They may nominate any number of potential directors and the Council will select from the pool a number of directors equal to the number of open seats.*)
+    * Each team representative must share their team's nominations and the reasoning for their nominations with the council's selected facilitator.
+    * The facilitator must publicly share the pool of all nominations and reasonings at least ten days before the final election meeting.
+        * This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting, this does not require restarting the 10-day period.
+
+### Election Meeting
+
+The election meeting is compromised of all members of the Leadership Council who will actively participate in the consensus process as well as the facilitator who will facilitate that process.
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format[^2] (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will share their set of nominees and the reasons for their change if that applies.
```

Comment:
It's not super clear to me what exactly is changing here - are council members withdrawing nominations? Altering reasons for/against? Withdrawing would seem odd to me given that the nomination was made by other people (potentially) and they're not here to hear arguments. Maybe we say that the arguments should be enough to decide in combination with the list of reasons for provided already, not sure...

Reply:
The idea here is to make room for people to change their minds based on the feedback they hear in the initial nomination round and to communicate how the information has influenced their disposition. At this point, you haven't made any proposal yet, so changing your set of nominees from one set of people to another doesn't inherently withdraw the prior set from consideration, if for whatever reason someone changed off of someone but everyone else changed too that person the final proposal could still be that person, or even if nobody changed to them and they weren't mentioned in the final round, though that would be strange and I would be really interested to hear the reasoning behind such a final proposal.

Reply:
> change their minds [...] made any proposal yet

I guess this is why I'm confused -- what are you changing your mind *on*? My impression is that we go in a round, say "hey, Joe would be great, they are involved in multiple parts of the project and are able to bridge gaps in understanding between folks coming from outside the project and inside".

Is the intent there that we're *also* saying something like "And I think Joe is better than Bob" in this round? And *that* ranking/choice is what is changing in the change round?

Reply:
Yeah, I think ultimately what the nomination + change rounds is to gather as much and as diverse of information as possible, this is also why we ask people to write down their nominations before hearing everyone talk, we want to hear your original perspective not have you go along with the group energy, because your idea might actually be a really unique and good one that nobody else considered. The change round is supposed to be a way to acknowledge that you received new information that has changed your mind. Maybe someone else's nomination mentioned that SoandSo actually has some amazing experience that's super relevant and you find that reasoning super compelling. In the change round you could acknowledge that and say "I'm changing my nomination to SoandSo because I forgot they're taking SuperRelevantTraining for roles just like this and I think this would be a perfect opportunity for them and the project to use and further practice those skills."

Reply:
I think this makes sense. Maybe the way to frame this is that the initial round is not *nominating* (which it sounds like you should really just do one person, if you switch to a single person in change round) and instead you list the *multiple* nominations your team made and reasons for (perhaps this is a "facilitator collects prior to the meeting?" - not clear why this is a sync round). Then we go through that list in rounds selecting the candidate each person believes is best and why they are best (vs. why they are good candidates) in the change rounds.

Does that roughly match your expectations, or am I missing some key elements? Mostly trying to repeat back my understanding rather than suggesting we make changes at this point - want to understand before (if) I ask for changes :)

Reply:
I really appreciate you looping it back for me, it's extremely helpful. I think there's still a disconnect for us to work out but we're making good progress.

> Maybe the way to frame this is that the initial round is not nominating and instead you list the multiple nominations your team made and reasons for.

So I think this accurately describes the current plan for the  [pre-meeting coordination](https://github.com/yaahc/leadership-council/blob/project-director-election-proposal/policies/project-director-election-process.md#pre-meeting-coordination) step. The teams first create a set of nominations that are then shared publicly with the other teams and representatives and hopefully reviewed prior to the meeting[^1]. The plan is to enter the meeting with a bunch of information from the pool of nominations and reasons from all the teams, plus a bunch of representatives who have participated in mini versions of the process with their subteams. Then the nomination round would have each representative suggest an exact set they'd propose based on the information they have available to them. The initial nomination the representatives make in the first round should include one name for each open position. It may be a subset of the nominations from their team, or it may utilize nominations from other teams if their reasons were compelling or if their original team didn't nominate enough people. Then you do the change round and the opportunity to tweak parts of what is essentially your own mini proposal based on the new information.

Hopefully, this helps. I think part of the confusion might be that I'm basing this proposal on a process designed to be fast and self-contained for things like filling roles within a team of people, not organization-wide decisions involving many teams recursively. I think we might have a little redundancy in the process, so it may end up being that the initial nomination round ends up functioning more like the change round of the async nomination round and that the change round in the election meeting itself is unnecessary. I'm hoping having extra rounds to speak where there's nothing to say is the safer failure mode and so I'm favoring redundancy, but I wouldn't be surprised if we further tweak the proposal after trying it out.

[^1]: And this is the part where I'm thinking maybe having the initial round be for listing the nominations and reasons in the pool is probably a good idea, but that's making me think three rounds, not two. Nomination pool sharing round, nomination proposal round (precise set of nominees based on open positions), proposal change round, final proposal + consent round (four rounds actually).

Reply:
Okay, I think we're generally on the same page on how this should work, now I'll need to spend some time rereading the text itself to make sure it aligns with my newfound understanding :)